tools/dds/dds-server/rs-dds-server.cpp

```diff
@@ -75,7 +77,7 @@ void start_streaming( std::shared_ptr< tools::lrs_device_controller > lrs_device
     header.format = static_cast< int >( vsp.format() );
     header.height = vsp.height();
     header.width = vsp.width();
-    dds_dev_server->set_image_header( stream_profile.stream_name(), header );
+    dds_dev_server->start_streaming( stream_profile.stream_name(), header );
```

Comment:
Looks strange that we have this flow:
```
dds_dev_server->start_streaming
lrs_device_controller->start_stream
```
no?

Reply:
I didn't even look at the controller. But that's on the "application" side. I can take a look later.